resources/Microsoft.Windows.Setting.Language/Microsoft.Windows.Setting.Language.psm1

```diff
@@ -0,0 +1,282 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+using namespace System.Collections.Generic
+
+function Test-LanguagePackAvailability {
+    param (
+        [Parameter(Mandatory = $true)]
+        [string]$Language
+    )
+
+    # Do not rely on Get-WindowsCapability as it requires elevation + it can take time to run
+    $languagePacks = Get-CimInstance -ClassName Win32_OperatingSystem | Select-Object -ExpandProperty MUILanguages
+
+    if ($null -eq $languagePacks) {
+        return $false
+    }
+
+    if ($languagePacks -notcontains $Language) {
+        return $false
+    }
+
+    return $true
+}
+
+function Test-UserLanguageList {
+    param (
+        [Parameter(Mandatory = $true)]
+        [string]$Language
+    )
+
+    $localeList = Get-WinUserLanguageList
+
+    if ($null -eq $localeList) {
+        return $false
+    }
+
+    if ($localeList.LanguageTag -notcontains $Language) {
+        return $false
+    }
+
+    return $false
+}
+
+function Get-LanguageList {
+    $languageList = Get-Language
```

Comment:
The tests are showing that this line is causing an error. You might need to import the `LanguagePackManagement` module at the top since you have several functions that depend on that module.